common/lib/xmodule/xmodule/js/src/video/035_video_accessible_menu.js

```diff
@@ -181,69 +190,69 @@ function () {
         return false;
     }
 
-    function _keyDownHandler(event) {
-        var KEY = $.ui.keyCode,
-            keyCode = event.keyCode,
-            target = $(event.currentTarget),
-            index;
-
-        if (target.is('a.a11y-menu-item-link')) {
-
-            index = target.parent().index();
-
-            switch (keyCode) {
-                // Scroll up menu, wrapping at the top. Keep menu open.
-                case KEY.UP:
-                    _previousMenuItemLink(this.menuItemsLinks, index).focus();
-                    break;
-                // Scroll down  menu, wrapping at the bottom. Keep menu
-                // open.
-                case KEY.DOWN:
-                    _nextMenuItemLink(this.menuItemsLinks, index).focus();
-                    break;
-                // Close menu.
-                case KEY.TAB:
-                    _closeMenu(this);
-                    // TODO
-                    // What has to happen here? In speed menu, tabbing backward
-                    // will give focus to Play/Pause button and tabbing
-                    // forward to Volume button.
-                    break;
-                // Close menu, give focus to button and change
-                // file type.
-                case KEY.ENTER:
-                case KEY.SPACE:
-                    this.button.focus();
-                    this.changeFileType.call(this, event);
-                    _closeMenu(this);
-                    break;
-                // Close menu and give focus to speed control.
-                case KEY.ESCAPE:
-                    _closeMenu(this);
-                    this.button.focus();
-                    break;
-            }
-            return false;
-        }
-        else {
-            switch(keyCode) {
-                // Open menu and focus on last element of list above it.
-                case KEY.ENTER:
-                case KEY.SPACE:
-                case KEY.UP:
-                    _openMenu(this);
-                    this.menuItemsLinks.last().focus();
-                    break;
-                // Close menu.
-                case KEY.ESCAPE:
-                    _closeMenu(this);
-                    break;
-            }
-            // We do not stop propagation and default behavior on a TAB
-            // keypress.
-            return event.keyCode === KEY.TAB;
-        }
-    }
+//     function _keyDownHandler(event) {
```

Comment:
@clrux as we discussed in person, I think that we should add support for navigating with the arrow keys in addition to tab key.  The aria authoring practices document describes this interaction.  If we want to emulate the behavior of the desktop (which we should) we should support arrowkey nav and tab.  Arrowkeys should behave just like tabs, so focus() should also be moved to the focusable next/prev focusable element.  Does anyone have any thought on what should happen when you reach the beginning or the end of the menu?  should it stop (bonk) or cycle to the other end?